app/models/concerns/subscribable.rb

```diff
@@ -12,4 +12,12 @@ def create_subscription
       Subscription.subscribe(user: user, to: self)
     end
   end
+
+  def current_subscription(user)
```

Comment:
I want to use this concern method too.
But I don't see the `current` in the name `current_subscription` 🤔 
What about:
`def user_subscription(user)`
`def subscription_for(user: user)`
`def find_subsbcription(user)`
...

Reply:
I vote for `subscription_for(user:)`, and I am open to suggestions, or to use `current_subscription`

Reply:
I'm going with `subscription_for`